custom_components/mqtt_vacuum_camera/camera.py

```
@@ -123,10 +124,10 @@ def __init__(self, coordinator, device_info):
         self.processor = CameraProcessor(self.hass, self._shared)
 
         # Listen to the vacuum.start event
-        self.hass.bus.async_listen("event_vacuum_start", self.handle_vacuum_start)
-        self.hass.bus.async_listen(
-            "mqtt_vacuum_camera_obstacle_coordinates", self.handle_obstacle_view
-        )
+        cancel = self.hass.bus.async_listen("event_vacuum_start", self.handle_vacuum_start)
+        cancel = self.hass.bus.async_listen(
```

Ruff (0.8.0)

Local variable `cancel` is assigned to but never used

Remove assignment to unused variable `cancel`

(F841)